packages/botonic-react/src/components/message.jsx

```diff
@@ -175,7 +175,7 @@ export const Message = props => {
           {isFromBot() && blob && (
             <div
               style={{
-                ...pointerStyles,
+                //...pointerStyles,
```

Comment:
should we remove the comment?
what is "pointer" here? can we find a better name?

Reply:
yes we can remove it.

Reply:
for the name, it doesn't come to my mind no others name for this. It was defined before this PR